### ISSUE TITLE
Ahead-of-time 'SortablejsModule' is not exported

### ISSUE THREAD

**Issue by craigsantos** [SortableJS/angular-sortablejs#34]
Hi,

I am having an issue including Sortablejs within a rollup ahead-of-time build. The error is **'SortablejsModule' is not exported by node_modules\angular-sortablejs\dist\index.js**
![image](https://cloud.githubusercontent.com/assets/15197892/24154579/7a45198c-0e49-11e7-832c-f103e6e6f61c.png)

rollup-config.js (abbreviated)
```JavaScript
import rollup      from 'rollup';
import commonjs    from 'rollup-plugin-commonjs';
import nodeResolve from 'rollup-plugin-node-resolve';
import uglify      from 'rollup-plugin-uglify';

export default {
entry: 'Scripts/Angular2/main-aot.js',
dest: 'Scripts/Angular2/dist/build.js',
sourceMap: true,
sourceMapFile: 'Scripts/Angular2/dist/build.js.map',
......
plugins: [
  nodeResolve({jsnext: true, module: true}),
  commonjs({
      include: ['node_modules/**']
  }),
  uglify()
]
};
```

app.module.ts (abbreviated)
```JavaScript
import { SortablejsModule } from 'angular-sortablejs';

@NgModule({
    imports: [SortablejsModule],
    .....
})
```

If I change the import line to import from `'angular-sortablejs/dist/src/sortablejs.module'`, the build succeeds but an error is thrown in the browser when the directive is used;
![image](https://cloud.githubusercontent.com/assets/15197892/24154977/717e7090-0e4a-11e7-9d40-8cc48795280f.png)
The same error occurs if I add this line to the commonjs config of rollup;
```JavaScript
namedExports: {  
          'node_modules/angular-sortablejs/dist/index.js' : ['SortablejsModule']
      }
```

Apologies if I am missing something obvious as I'm fairly new to this.

**Comment by smnbbrv** [comment]
Hi @craigsantos 

it's really hard to follow all the build tools and this is (shame on me) the first time I hear of rollup.

It would be helpful if you can adapt the angular-sortablejs that it is covering rollup as well. All PRs are warmly welcomed.

**Comment by smnbbrv** [comment]
should be fixed with 2.0.0

**Comment by craigsantos** [comment]
Thanks. I have been trying to work on a pull request to fix this but will try out the change first!
